CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ IMPROVEMENTS:
 * Control-Plane
   * Add support for the annotation `consul.hashicorp.com/use-proxy-health-check`. When this annotation is used by a service, it configures a readiness endpoint on Consul Dataplane and queries it instead of the proxy's inbound port which forwards requests to the application. [[GH-1824](https://github.com/hashicorp/consul-k8s/pull/1824)], [[GH-1841](https://github.com/hashicorp/consul-k8s/pull/1841)]
   * Add health check for synced services based on the status of the Kubernetes readiness probe on synced pod. [[GH-1821](https://github.com/hashicorp/consul-k8s/pull/1821)]
+* CLI:
+  * Add `consul-k8s proxy log podname` command for displaying and mofidying Envoy log levels for a given Pod. [GH-1844](https://github.com/hashicorp/consul-k8s/pull/1844), [GH-1849](https://github.com/hashicorp/consul-k8s/pull/1849), [GH-1864](https://github.com/hashicorp/consul-k8s/pull/1864)
```

Comment:
This is great!

Reply:
If these all merge into a single feature branch, I would just include the one feature branch that targets `main` instead of all 3. Just for tidiness.

Reply:
so each of these PR's is targeting `main`, right now this PR is targeting the `set-envoy...` branch because it's dependent on some of the work in there

Reply:
You have a spelling mistake 'mofidying'

Reply:
nice catch! lemme fix that